shell/browser/electron_browser_client.cc

```diff
@@ -462,6 +462,11 @@ void ElectronBrowserClient::AppendExtraCommandLineSwitches(
     base::FilePath child_path;
     base::FilePath program =
         base::MakeAbsoluteFilePath(command_line->GetProgram());
+    if (program.empty()) {
+      PLOG(ERROR) << "Failed to make absolute path for program: "
+                  << command_line->GetProgram();
+      exit(1);
```

Comment:
if we exit here, we'd probably want to do something similar to other early exits in this function:

```cpp
CHECK(!program.empty())
          << "Failed to make absolute path for program: "
          << command_line->GetProgram();
```

is there any scena